### PR TITLE
Report errors from resource loading and apply unexpected input behavior more sparingly

### PR DESCRIPTION
**Pull request by jarrettranderson**

### CODE REVIEW

File: ipv-core/src/main/java/com/ibm/whc/deid/models/LatitudeLongitude.java [hunk 1]
```diff
@@ -84,6 +61,39 @@ public LatitudeLongitude(double latitude, double longitude, LatitudeLongitudeFor
...
+        throw new IllegalArgumentException(
+            Messages.getMessage(LogCodes.WPH1010E, latitude, "latitude"));
+      }
+      try {
+        lon = Double.parseDouble(longitude);
+      } catch (NumberFormatException e) {
+        throw new IllegalArgumentException(
+            Messages.getMessage(LogCodes.WPH1010E, longitude, "longitude"));
+      }
+      value = new LatitudeLongitude(lat, lon, LatitudeLongitudeFormat.DECIMAL);
```
[the remark this answers is not in the document]
Comment (jarrettranderson) [reply ipv-core/src/main/java/com/ibm/whc/deid/models/LatitudeLongitude.java:92]: I believe we will get that for "free" when this line enters the constructor and eventually hits line 60

File: ipv-core/src/test/resources/localization/test.races.bad.csv [hunk 2]
```diff
@@ -0,0 +1,18 @@
+American Indian or Alaska Native
+Asian
+Black or African American
+Hispanic or Latino
+Native Hawaiian or Other Pacific Islander
+White
```
Comment (DenisRicard) [inline ipv-core/src/test/resources/localization/test.races.bad.csv:6]: Should probably replace white with caucasian

Comment (DenisRicard) [reply ipv-core/src/test/resources/localization/test.races.bad.csv:6]: Actually, loking at the HL7 org, it doesn't even have caucasian so what's there is probably fine. 

https://www.hl7.org/fhir/v3/Ethnicity/cs.html

**Review by DenisRicard — APPROVED** [review]
All comments answered. Approving

File: ipv-core/src/test/resources/localization/test.races.bad.csv (hunk 2 above)
Comment (jarrettranderson) [reply ipv-core/src/test/resources/localization/test.races.bad.csv:6]: "white" was the value used in the US federal register entry from which we got the race values.  These are excerpts from the shipped file.

**Pull request merged**